### ISSUE TITLE
SLF4j configuration warning on project build

### ISSUE THREAD

**Issue by maxcom** [webjars/webjars-play#49]
Sometimes I see this warning on project build:

{code}
Information:Play 2 Compiler:  Compiling 9 Scala sources to /home/maxcom/git-checkout/xxxx/target/scala-2.11/test-classes...
Information:Play 2 Compiler: SLF4J: The following set of substitute loggers may have been accessed
Information:Play 2 Compiler: SLF4J: during the initialization phase. Logging calls during this
Information:Play 2 Compiler: SLF4J: phase were not honored. However, subsequent logging calls to these
Information:Play 2 Compiler: SLF4J: loggers will work as normally expected.
Information:Play 2 Compiler: SLF4J: See also http://www.slf4j.org/codes.html#substituteLogger
Information:Play 2 Compiler: SLF4J: org.webjars.CloseQuietly
Information:Play 2 Compiler: [success] Total time: 21 s, completed 03.03.2015 13:34:49
{code}

**Comment by jamesward** [comment]
Hmmm...  Not sure what is causing this.  Maybe @huntc has an idea.

**Comment by markusjura** [comment]
Happens for me in Play 2.4 as well: http://www.slf4j.org/codes.html#substituteLogger

**Comment by jirihelmich** [comment]
Happened for me in Play 2.5 as well, however, I'm not using any WebJar in my project. The stack trace is the same. I had a look around and the problem went away when I removed all the initial play web plugins:

```


addSbtPlugin("com.typesafe.sbt" % "sbt-coffeescript" % "1.0.0")

addSbtPlugin("com.typesafe.sbt" % "sbt-less" % "1.1.0")

addSbtPlugin("com.typesafe.sbt" % "sbt-jshint" % "1.0.4")

addSbtPlugin("com.typesafe.sbt" % "sbt-rjs" % "1.0.8")

addSbtPlugin("com.typesafe.sbt" % "sbt-digest" % "1.1.1")

addSbtPlugin("com.typesafe.sbt" % "sbt-mocha" % "1.1.0")

addSbtPlugin("org.irundaia.sbt" % "sbt-sassify" % "1.4.6")
```

**Comment by jamesward** [comment]
Yeah, so this is something from sbt-web so @huntc will need to look at this when he has time.
